lxd/images.go

```diff
@@ -1525,7 +1568,7 @@ func doImageGet(db *db.Cluster, project, fingerprint string, public bool) (*api.
 	return imgInfo, nil
 }
 
-func imageValidSecret(fingerprint string, secret string) bool {
+func imageValidSecret(fingerprint string, secret string) (*operations.Operation, bool) {
```

Comment:
This change appears unrelated to this commit, can you split it out or move it to a more suitable commit?